crates/core/src/db/models.rs

```diff
@@ -188,6 +188,24 @@ pub struct Storefront<'a> {
     pub banner_url: Option<Cow<'a, str>>,
 }
 
+/// Join of metadata and metadata_json for an NFT
+#[derive(Debug, Clone, Queryable)]
+pub struct Nft {
+    /// Table metadata
```

Comment:
```suggestion
    // Table metadata
```
Rust counts comments with three slashes as syntactically important and converts them to `#[doc = "..."]` attributes